apps/condo/domains/property/components/panels/Builder/BuildingPanelView.tsx

```
@@ -332,143 +391,60 @@ export const PropertyMapView: React.FC<IPropertyMapViewProps> = ({ builder, refr
+                                                    if (floorInfo && floorInfo.units.length) {
+                                                        return (
+                                                            <UnitButtonWithContactInfo
+                                                                floorInfo={floorInfo}
+                                                                property={property}
+                                                                getContacts={getContacts}
+                                                                contactsLoading={contactsLoading}
+                                                                contacts={contactsData?.contacts}
+                                                                contactsError={contactsError}
+                                                                setModalOpenedUnit={setModalOpenedUnit}
+                                                                toggleUnitModal={toggleUnitModal}
+                                                            />
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)